wurst/heroes/heroes/DemonLord/Obliterator.wurst

```diff
@@ -0,0 +1,56 @@
+package Obliterator
+import AbilityTooltipGenerator
+import ClosureForGroups
+import ClosureTimers
+import CreepAggro
+import DemonLordIds
+import HashMap
+import Icons
+
+constant IntLevelClosure BONUS_STRENGTH = _ -> 50
+constant IntLevelClosure CREEP_AGGRO = _ -> 200
+constant RealLevelClosure CREEP_AGGRO_RANGE = _ -> 400.
+constant RealLevelClosure BONUS_HEALTH_THRESHOLD = _ -> .5
+
+constant DEMON_LORD_STR_BONUSES = new HashMap<unit, int>()
+
+function int.toCustomUnitTypeString() returns string
+  return "custom_" + this.toRawCode()
+
+init
+  doPeriodically(1.) _ ->
```

Comment:
Why is this running globally like this? Heroes can only exist once.

Reply:
You didn't answer why this runs periodically. Are you evaluating the "buff" here? then you should use a proper visual buff using frentity, which should be better anyway.